backend/src/api/events.py

```
@@ -171,6 +183,135 @@ async def get_event_dashboard_stats(
+# Conflict Detection & Scoring Endpoints (must be before /{guid})
+# ============================================================================
+
+
+@router.get(
+    "/conflicts",
+    response_model=ConflictDetectionResponse,
+    summary="Detect scheduling conflicts",
+    description="Returns conflict groups with scored events for the specified date range",
+)
+async def detect_conflicts(
+    start_date: date = Query(..., description="Start of date range (inclusive)"),
```

Ruff (0.15.0)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)